tcr-engine/events/test_run_information.go

```diff
@@ -0,0 +1,22 @@
+package events
+
+// TestRunInformation is the structure containing information of the test run
+type TestRunInformation struct {
+	TotalTestsRan   int
```

Comment:
There is redundancy between the structure name and the field names. For example, TestRunInformation.TotalTestsRan could be simply called TestRunInformation.Total, TestRunInformation.TestsPassed could be called TestRunInformation.Passed, etc.